packages/react-native/Libraries/Network/RCTNetworkTask.h

```diff
@@ -35,7 +35,7 @@ typedef NS_ENUM(NSInteger, RCTNetworkTaskStatus) {
 @property (nonatomic, copy) RCTURLRequestResponseBlock responseBlock;
 @property (nonatomic, copy) RCTURLRequestProgressBlock uploadProgressBlock;
 
-@property (nonatomic, readonly) RCTNetworkTaskStatus status;
+@property (atomic, readonly) RCTNetworkTaskStatus status;
```

Comment:
The atomic attribute doesn't really matter here, since we use a custom getter anyway.

Reply:
I see. However, as for the API surface of the class, won't it make sense to convey it is indeed atomic even though it is making use of other mechanisms than objc's synchronize?